docs/sources/auth/auth-proxy.md

```diff
@@ -294,3 +297,12 @@ curl -H "X-WEBAUTH-USER: leonard" -H "X-WEBAUTH-GROUPS: lokiteamOnExternalSystem
 With this, the user `leonard` will be automatically placed into the Loki team as part of Grafana authentication.
 
 [Learn more about Team Sync]({{< relref "auth/team-sync.md" >}})
+
+
+## Login token & session cookie
+
+If you want to combine auth proxy with the normal login session token & cookie enable the setting `grant_login_token`.
+With that option enabled requests to `/login` that have auth proxy headers added will create a new session token for the
+user (and redirect user). This means that only /login route needs to go through auth proxy. Use settings `login_maximum_inactive_lifetime_days`
```

Comment:
Should explain something along the lines: In this setup the auth proxy is responsible for authenticating the user, but Grafana will keep track of the session and keep the user logged in why the auth proxy only needs to provide headers for the /login route. 

Would be good to link to `[Login and short-lived tokens](/auth/overview/#login-and-short-lived-tokens) which has all the details about it.